include/mbedtls/x509_crt.h

```diff
@@ -175,19 +175,34 @@ int mbedtls_x509_crt_parse_der( mbedtls_x509_crt *chain, const unsigned char *bu
                         size_t buflen );
 
 /**
- * \brief          Parse one or more certificates and add them
- *                 to the chained list. Parses permissively. If some
- *                 certificates can be parsed, the result is the number
- *                 of failed certificates it encountered. If none complete
- *                 correctly, the first error is returned.
+ * \brief          Parse one DER-encoded or multiple concatenated PEM-encoded
```

Comment:
Nit picking: I believe this implies that this function accepts either one DER encoded certificate or multiple concatenated PEM encoded certificate. It doesn't suggest that a single PEM encoded certificate

Reply:
@RonEld I don't understand -- what's what the documentation is saying, isn't it?

Reply:
Or are you saying that it doesn't state clearly enough that a single PEM CRT is ok, too?

Reply:
>Or are you saying that it doesn't state clearly enough that a single PEM CRT is ok, too?

Yes

Reply:
Will fix it.